api/configs/deploy/__init__.py

```diff
@@ -17,11 +17,6 @@ class DeploymentConfig(BaseSettings):
         default=False,
     )
 
-    TESTING: bool = Field(
```

Comment:
I have observed that the key is still being used in the test. Is it expected?

Reply:
I found no reference of `dify_config.TESTING`. Did I miss anything?

Reply:
The "TESTING" key is used by `app.config` in tests, do we still need it? In `test_dify_config` L74

Reply:
OK, let me remove test_dify_config L74.

Reply:
Another "TESTING" in `conftest.py` L13

Reply:
Resolved.